src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,113 @@
 package core.basesyntax;
 
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROWTH_INDEX = 1.5;
+    private Object[] elements;
+    private int size;
+
+    public ArrayList() {
+        elements = new Object[DEFAULT_CAPACITY];
+    }
+
     @Override
     public void add(T value) {
-
+        if (size == elements.length) {
+            arrayIncrease();
+        }
```

Comment:
this code is unnecessary, because you have `if (minCapacity > elements.length)` inside `ensureCapacity` method
```suggestion
        ensureCapacity(size + 1);
```